---
name: Jason E. Stajich, Ph.D.
startdate: YYYY-MM-DD
enddate: YYYY-MM-DD
image: /static/img/members/Stajich_2019_2.jpg
# altimage: /static/img/members/Stajich_2019_2.jpg
position: Principal Investigator
#current:
pronouns: he/him/his
email: jason.stajich (at) ucr.edu
scholar: j3iaoR8AAAAJ
website:
twitter: stajichlab
linkedin: 
github: stajichlab
orcid: 0000-0002-7591-0020
description: "I am interested in the process and mechanisms of evolution. I study this primarily in fungi using comparative, computational, and experimental tools.  We utilize genome and RNA sequencing, sequence analysis, molecular evolution, and phylogenetics, and molecular biology tools to explore the functions of genes or genomic regions identified by analyses to be involved in processes we study.


Most of our work is focused in the [zygomycete](http://zygolife.org/) and zoosporic [chytrid](http://chytrids.org/) fungi (fungi that move!). We also have collaborative projects and interests in *Aspergillus*, *Fusarium*, *Coccidioides*, and *Clavispora lusitaniae*. The lab is increasingly moving towards questions that relate to symbioses with new projects on fungal-bacteria antagonism and on the biological symbioses that occur among fungi, algae, bacteria in desert Biological crusts. I also have a new interest in extremophile fungi and working on projects to understand the halophilic *[Hortaea](https://www.ncbi.nlm.nih.gov/bioproject/356640) werneckii* and endolithic [Antarctic fungi](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA342238) through genome sequencing and laboratory experiments.


I am involved in several fungal genome projects including co-leading the [1000 fungal genomes project](http://1000.fungalgenomes.org/) with the [JGI](http://www.jgi.doe.gov/) and the [ZyGoLife](http://zygolife.org/) project.


My interests are also in the evolution of multicellular forms and regulation of development in fungi.  I think understanding how differential gene regulation is established can help learn more about the mechanisms of cell type differentiation.  We are also studying the cell wall to understand how innovations in the cell wall and dimorphism impact interactions between pathogenic fungi and hosts they infect. These different projects seek to provide new insight into the big picture of how the complexity of life evolved and how host and pathogen interactions co-evolved.


To address this work we also need tools to sift and mine the gigantic datasets that genomics can produce. I have focused on building tools for comparative and computational analyses of genomes including work on the [BioPerl](http://bioperl.org/) and [Gbrowse](http://www.gmod.org/wiki/Gbrowse) projects and the development of open source software for bioinformatics and life sciences research through the [Open Bioinformatics Foundation](http://www.open-bio.org/).

The lab is also focused on the development of databases for [fungal genome data](http://fungalgenomes.org/) to make the genome and functional information more available. I also blog about interesting findings in fungal, microbial, and genome research and share protocols and coordinate projects through a [wiki site](http://fungalgenomes.org/wiki)."
---
